### PR TITLE
HDDS-5178. Update project information of Contribution guideline

### PR DESCRIPTION
**Pull request by sky76093016**
## What changes were proposed in this pull request?

Update link of wiki pages.

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-5178

## How was this patch tested?

No test.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ We welcome contributions of:
  * **Documentation Improvements**: You can submit improvements to either:
      * Ozone website. Instructions are here: [Modifying the Ozone Website](https://cwiki.apache.org/confluence/display/HADOOP/Modifying+the+Ozone+Website)
      * Developer docs. These are markdown files [checked into the Apache Ozone Source tree](https://github.com/apache/ozone/tree/master/hadoop-hdds/docs/content).
- * The [wiki pages](https://cwiki.apache.org/confluence/display/HADOOP/Ozone+Contributor+Guide): Please contact us at ozone-dev@hadoop.apache.org and we can provide you write access to the wiki.
+ * The [wiki pages](https://cwiki.apache.org/confluence/display/OZONE/Contributing+to+Ozone): Please contact us at ozone-dev@hadoop.apache.org and we can provide you write access to the wiki.
```
Comment (bharatviswa504) [inline CONTRIBUTING.md:18]: Can you also update ozone-dev@hadoop.apache.org -> dev@ozone.apache.org

Comment (sky76093016) [reply CONTRIBUTING.md:18]: Sure.

**Review by ayushtkn — COMMENTED** [review]
There is a mention of `hadoop-ozone` in the same file, Can you correct the repo name it is apache/ozone now
`Fork apache/hadoop-ozone github repository (first time)`

**Comment by sky76093016** [comment]
@ayushtkn I will modify it right away.

**Review by bharatviswa504 — APPROVED** [review]
+1 LGTM.
Thank You @sky76093016 for the fix.

**Comment by cxorm** [comment]
Thanks @sky76093016 for the work and @bharatviswa504, @ayushtkn for the reviews.

The Jira is for updating link in the first but we updated project information during careful reviews,
so I suggest that we could change the title of the Jira if you don't mind. 
(For example, `Update project information of Contribution guideline`)

**Comment by sky76093016** [comment]
@cxorm Thanks for your suggestion.
Of course, it looks like a good title.You can modify the title of jira.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ We welcome contributions of:
  * **Documentation Improvements**: You can submit improvements to either:
      * Ozone website. Instructions are here: [Modifying the Ozone Website](https://cwiki.apache.org/confluence/display/HADOOP/Modifying+the+Ozone+Website)
```
Comment (cku328) [inline CONTRIBUTING.md:16]:
```suggestion
     * Ozone website. Instructions are here: [Modifying the Ozone Website](https://cwiki.apache.org/confluence/display/OZONE/Modifying+the+Ozone+Website)
```
This link also needs to be updated `HADOOP` -> `OZONE`

**Review by cku328 — CHANGES_REQUESTED** [review]
Thanks @sky76093016 for improving this guideline.

In the `Who To Contact` section of the same file, can you also update the dev-mailing list address and weekly meetings link?

image

File: CONTRIBUTING.md (hunk 2 above)
Comment (sky76093016) [reply CONTRIBUTING.md:16]: Sure.

**Review by cku328 — APPROVED** [review]
LGTM. 
Thanks @sky76093016 for the update.

**Comment by bharatviswa504** [comment]
Thank you @sky76093016 for the contribution, 
@ayushtkn, @cku328, and @cxorm for the review.

**Pull request merged**